### PR TITLE
deployment e2e: print out the error

### PR DESCRIPTION
**Pull request by nikhiljindal**

### CODE REVIEW

**Comment by nikhiljindal** [comment]
cc @janetkuo @bgrant0607

File: test/e2e/deployment.go [hunk 1]
```diff
@@ -194,7 +194,7 @@ func stopDeployment(c *clientset.Clientset, oldC client.Interface, ns, deploymen
 		}
 		return false, nil
 	}); err != nil {
-		Failf("Err : %s\n. Failed to remove deployment %s pods : %+v", deploymentName, pods)
+		Failf("Err : %s\n. Failed to remove deployment %s pods : %+v", err, deploymentName, pods)
```
Comment (bgrant0607) [inline test/e2e/deployment.go]: Good catch. Is %s the right format here? What about aggregated errors, etc.?

**Comment by bgrant0607** [comment]
LGTM

**Comment by bgrant0607** [comment]
Merging to help debug flakes.

File: test/e2e/deployment.go (hunk 1 above)
Comment (nikhiljindal) [inline test/e2e/deployment.go]: Yes, we print all our errors with %s

**Pull request merged**